src/index.js

```diff
@@ -11,7 +11,11 @@
 
 import { Application } from 'stimulus';
 import { definitionsFromContext } from 'stimulus/webpack-helpers';
-import symfonyControllers from '@symfony/controllers';
+
+// this import is be replaced by the NormalModuleReplacementPlugin
+// to point to the real controllers.json file
+// See create-controllers-module.js
+import symfonyControllers from './webpack/loader!../controllers-placeholder.json';
```

Comment:
Is the loader necessary here? If the NormalModuleReplacementPlugin pass before modules, we could have a nicer syntax here, like `@symfony/controllers` :)

Reply:
Per your recommendation, `NormalModuleReplacementPlugin` is gone in favor of an alias :).

So, the loader IS needed here and I think it's helpful: it helps clarify (for a curious reader) that we are not parsing this as JSON, but doing something trickier.